.eslintrc.json

```diff
@@ -54,6 +60,8 @@
         "react/jsx-filename-extension": "error",
         "react/prefer-stateless-function": "warn",
         "react/sort-comp": "warn",
-        "react/jsx-max-props-per-line": [1, { "when": "multiline" }]
+        "react/jsx-max-props-per-line": [1, { "when": "multiline" }],
+        "import/namespace": "off",
+        "import/default": "off"
```

Comment:
I would go for one eslint config. Otherwise you need to run eslint twice I think. E.g. CLI tool, editor integrations. I can only set one eslint config in Webstorm for example.

With the jest/globals there are two options:
1. Keep it in the eslint config and have the tradeof that the testing globals are not reported as undefined in non-testing files.
2. Remote it in the eslint config and add `/* eslint-env jest */` in all test files.

I use option 2 in the main project that I'm working on.

For the import rules I don't really see why those need to be different for test files. Maybe we can resolve the underlying issue.